### DISCUSSION TITLE
Permission Error: [Error 13] Permission denied while creating a task

### DISCUSSION THREAD

**Discussion by keshavadk** [cvat-ai/cvat#4320] [Q&A]
Created a CVAT set up in my local machine . 
All the services including Frontend , Backend, worker and opa are running.

After logging into the tool, i am able to create the "Project", but when i create a "task", the task creation is pending as shown in the image below in the browser.

![MicrosoftTeams-image (4)](https://user-images.githubusercontent.com/16021947/153390465-5d4a220d-2542-4ffc-bca8-d7cffc15e755.png)

Also, if we check the Worker terminal, the service will be stopped automatically and it gives an error saying 

Permission Error: [Error 13] Permission denied:  

![MicrosoftTeams-image (3)](https://user-images.githubusercontent.com/16021947/153390494-705175db-7380-4066-950a-bdcf03eaf7f1.png)

Any hint ,please ?

**Comment by azhavoro** [comment]
Have you checked the directory/file permissions? CVAT itself doesn't change any of file/folder permissions.
All cvat process are running under current user, so make sure you have correct filesystem permissions

**Reply by keshva23** [reply]
@azhavoro  Ok we will check. Thank you for the reply.
